<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Grath</class>
 <widget class="QDialog" name="Grath">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>518</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>558</width>
    <height>518</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>558</width>
    <height>518</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:#000;
border-color: #000;</string>
  </property>
  <widget class="QCustomPlot" name="Grathic" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>531</width>
     <height>330</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>531</width>
     <height>330</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>531</width>
     <height>330</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>533</width>
     <height>172</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Waree</family>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Область определения (х)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Waree</family>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>От</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="grathXMin">
          <property name="font">
           <font>
            <family>Waree</family>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 5px solid rgb(211, 215, 207);
background-color:rgb(255, 255, 255);
border-radius: 5px;</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="maxLength">
           <number>15</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>Waree</family>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>До</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLineEdit" name="grathXMax">
          <property name="font">
           <font>
            <family>Waree</family>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 5px solid rgb(211, 215, 207);
background-color:rgb(255, 255, 255);
border-radius: 5px;</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="maxLength">
           <number>15</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Waree</family>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Область значения (у)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>Waree</family>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>От</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="grathYMin">
          <property name="font">
           <font>
            <family>Waree</family>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 5px solid rgb(211, 215, 207);
background-color:rgb(255, 255, 255);
border-radius: 5px;</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="maxLength">
           <number>15</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Waree</family>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>До</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLineEdit" name="grathYMax">
          <property name="font">
           <font>
            <family>Waree</family>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 5px solid rgb(211, 215, 207);
background-color:rgb(255, 255, 255);
border-radius: 5px;</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="maxLength">
           <number>15</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="buildGrath">
        <property name="font">
         <font>
          <family>Waree</family>
          <pointsize>-1</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    padding: 10px 10px;
    font-size: 16px;
    text-align: center;
    color: #000000;
    background-color: #e5e1e0;
    border: 2px solid #ccc4c3;
    border-radius: 15px;
}

QPushButton:hover {
    background-color: #b3a7a6;
    border-color: #b3a7a6;
}

QPushButton:pressed {
    background-color: #6c5755;
    border-color: #6c5755;
}</string>
        </property>
        <property name="text">
         <string>Построить график</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Waree</family>
        <pointsize>7</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>*Область значения и определения ограничена от - 1 000 000 до 1 000 000; Масштаб 1:1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
